docs/app/Examples/elements/Input/Variations/InputExampleActionLabeledButton.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { Input } from 'semantic-ui-react'
+
+const InputExampleActionLabeledButton = () => (
+  <Input
+    action={{ color: 'teal', label: 'right', icon: 'copy', content: 'Copy' }}
```

Comment:
The prop `label` is for defining the label, the prop `labelPosition` is for placing it on the left/right.  This should be `labelPosition: 'right'`.